kernel/cClosure.mli

```diff
@@ -179,7 +187,7 @@ val unfold_projection : 'a infos -> Projection.t -> stack_member option
    [create_clos_infos], inject the term to reduce with [inject]; then use
    a reduction function *)
 
-val inject : constr -> fconstr
+val inject : constr puniverses -> fconstr
```

Comment:
The type of this function should probably not have been changed. In the same way that it assumes the identity substitution for variables it should assume no universe substitution.